src/components/badges/BadgeSection.tsx

```diff
@@ -0,0 +1,28 @@
+import { BadgeSectionTitle } from 'components/Text'
+import { ComponentChildren } from 'preact'
+import ChildrenProp from 'models/ChildrenProp'
+import classnames, {
+  display,
+  gap,
+  gridAutoRows,
+  gridTemplateColumns,
+} from 'classnames/tailwind'
+
+const badgesList = classnames(
+  display('grid'),
```

Comment:
We use same styles for in `List.tsx`, maybe it's better to reuse them?

Reply:
They are no longer in `List.tsx` 🤔